app/models/outgoing_message.rb

```diff
@@ -261,6 +261,13 @@ def mail_server_logs
     end
   end
 
+  def delivery_status
+    mail_server_logs.
+      map { |log| log.line(:decorate => true).delivery_status }.
+        compact.
+          last
```

Comment:
Seems to have been fine when looking through some logs for https://github.com/mysociety/alaveteli/pull/3039